spec/index.bs

```diff
@@ -1741,13 +1751,43 @@ enum GPUAddressMode {
 };
 </script>
 
+<dl dfn-type="enum-value" dfn-for=GPUAddressMode>
+    : <dfn>"clamp-to-edge"</dfn>
+    ::
+        Texture coordinates are clampled between 0.0 and 1.0, inclusive.
+
+    : <dfn>"repeat"</dfn>
+    ::
+        Texture coordinates wrap to the other side of the texture.
+
+    : <dfn>"mirror-repeat"</dfn>
+    ::
+        Texture coordinates between -1.0 and 1.0, inclusive, are mirrored across the axis.
+        Outside -1.0 and 1.0, the texture coordinates wrap to the other side of the texture.
```

Comment:
I don't think this is correct. What about, say, coordinate -2.5 ? We need to say something like: the texture is mirror expanded to -1.0 to 1.0 range, and that one is expanded. For example, this is how D3D12 describes it:
> Flip the texture at every (u,v) integer junction.
For u values between 0 and 1, for example, the texture is addressed normally; between 1 and 2, the texture is flipped (mirrored); between 2 and 3, the texture is normal again; and so on.